src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -182,6 +188,32 @@ private void applyStringFilter(String filterString) {
         }
     }
 
+    private void indicatePanelLoading(RepoOpeningEvent e) {
+        togglePanelLoading(e.isPrimaryRepo, e.repoId, true);
+    }
+
+    private void unindicatePanelLoading(RepoOpenedEvent e) {
+        togglePanelLoading(e.isPrimaryRepo, e.repoId, false);
+    }
```

Comment:
Actually, there are two different events. Repo OpenING Event and Repo OpenED Event. That's why I needed two different methods to handle them